lib/bokken_web/router.ex

```diff
@@ -50,6 +50,9 @@ defmodule BokkenWeb.Router do
       get "/files", FileController, :index
     end
 
+    resources "/skills", SkillController, except: [:new, :edit]
+    resources "/user_skills", UserSkillController, except: [:new, :edit, :update]
```

Comment:
The resource `/skills` should also be inside the resource `/mentors` and the resource `/ninjas`, no need for the `/user_skills`.